Tools/Scripts/webkitpy/w3c/test_exporter.py

```diff
@@ -352,13 +352,22 @@ def do_export(self):
             if lint_errors:
                 _log.error(f'The wpt linter detected {lint_errors} linting error(s). Please address the above errors before attempting to export changes to the web-platform-test repository.')
                 self.delete_local_branch(is_success=False)
-                return
+                return 1
+
+        if self._options.dry_run:
+            _log.info('Skipping pushing to remote since this is a dry run')
+            self.delete_local_branch(is_success=True)
```

Comment:
Can you split this into a separate change?

Reply:
Oh, no, this isn't a change; it's just a move away from the convoluted logic below. Sorry; I didn't expand the diff enough!